### ISSUE TITLE
Replication events: info.doc is info.docs

### ISSUE THREAD

**Issue by gr2m** [pouchdb/pouchdb#3830]
I thinkt there is an error in the docs, in the `Replication events` section.

```js
db = new PouchDB('mydb')
db.put({_id: '1'})
db.replicate.to('mydb2', { create_target: true, include_docs: true })
.on('change', function(change){
  // change.doc is undefined
  // change.docs is [{_id: '1', _rev: '1-...'}]
})
```

**Comment by nolanlawson** [comment]
Yep, I think you're right. Pull request? :D

**Comment by gr2m** [comment]
It was actually @zoepage who run into it. I'll leave the honour to her ;)

By the way, can you tell in what circumstances `change.docs` could have more than one doc? We were wondering about that today

**Comment by daleharvey** [comment]
@gr2m @zoepage Where are you seeing this? I couldnt actually see it documented at all (https://github.com/pouchdb/pouchdb/pull/4123)

**Comment by daleharvey** [comment]
Ah ok I found it

And as for having more than one doc, that should happen mostly when doing a new replication, the `docs` are all the docs that were written during a single batch, the first 1 or 2 batches are always individual docs, then the batches start filling up and you will get multiple documents

**Comment by nolanlawson** [comment]
should be fixed by 53cb41fe3c16031d7eac1c65fc7b0e71adc820ca
